examples/tensorflow/common/models.py

```diff
@@ -199,24 +199,28 @@ def _depth(v, divisor=8, min_value=None):
 def _se_block(inputs, filters, se_ratio, prefix):
     x = layers.GlobalAveragePooling2D(name=prefix + 'squeeze_excite/AvgPool')(
         inputs)
-    if backend.image_data_format() == 'channels_first':
-        x = layers.Reshape((filters, 1, 1))(x)
-    else:
-        x = layers.Reshape((1, 1, filters))(x)
-    x = layers.Conv2D(
+    #if backend.image_data_format() == 'channels_first':
+    #    x = layers.Reshape((filters, 1, 1))(x)
+    #else:
+    #    x = layers.Reshape((1, 1, filters))(x)
```

Comment:
Is it a temporary change? It seems we should handle it without commenting

Reply:
Yes, it's only to test if nncf TF will work with se blocks boiled on dense layers instead of conv2d. PR is really raw by now